pkg/didcomm/protocol/introduce/service_test.go

```diff
@@ -367,81 +425,24 @@ func TestService_Accept(t *testing.T) {
 	defer ctrl.Finish()
 
 	storageProvider := storageMocks.NewMockProvider(ctrl)
-	storageProvider.EXPECT().OpenStore(Introduce).Return(nil, nil)
+	storageProvider.EXPECT().OpenStore(introduce.Introduce).Return(nil, nil)
 
-	provider := mocks.NewMockProvider(ctrl)
+	provider := introduceMocks.NewMockProvider(ctrl)
 	provider.EXPECT().StorageProvider().Return(storageProvider)
 	provider.EXPECT().OutboundDispatcher().Return(nil)
-	provider.EXPECT().Service(didexchange.DIDExchange).Return(mocks.NewMockForwarder(ctrl), nil)
+	provider.EXPECT().Service(didexchange.DIDExchange).Return(introduceMocks.NewMockForwarder(ctrl), nil)
 
-	svc, err := New(provider)
+	svc, err := introduce.New(provider)
 	require.NoError(t, err)
 
 	defer stop(t, svc)
 
 	require.False(t, svc.Accept(""))
-	require.True(t, svc.Accept(ProposalMsgType))
-	require.True(t, svc.Accept(RequestMsgType))
-	require.True(t, svc.Accept(ResponseMsgType))
-	require.True(t, svc.Accept(AckMsgType))
-	require.True(t, svc.Accept(ProblemReportMsgType))
-}
-
-func Test_stateFromName(t *testing.T) {
```

Comment:
did these tests get moved?

Reply:
Reverted! FYI: This test will not increase coverage. This is why I thought it was redundant. But anyway better to have more tests :)